Automattic/wp-calypso
# Replace No Ads upsell Nudge with Premium plan nudge

Implements the request defined in pau2Xa-4vo-p2 to remove the No-Ads addon.

#### Proposed Changes

* In Tools > Marketing, verify that the No Ads addon upsell is replaced with a Premium plan upsell.
* Clicking the Premium upsell should take to the `/plans` page with the "Remove WordPress.com Ads" feature highlighted.

**BEFORE**

<img width="655" alt="Screenshot 2022-10-26 at 1 13 41 PM" src="https://user-images.githubusercontent.com/1269602/197965272-ec682e46-61f0-4883-9c23-4a0762582d71.png">

**AFTER**

<img width="773" alt="Screenshot 2022-10-26 at 12 50 30 PM" src="https://user-images.githubusercontent.com/1269602/197965333-cb9270af-768b-486f-9dc8-2d09a748dbb5.png">

Clicking the nudge will lead to /plans with the feature highlighted:

<img width="1431" alt="Screenshot 2022-10-26 at 1 24 32 PM" src="https://user-images.githubusercontent.com/1269602/197967611-bb1bee3a-062d-43c5-898b-b791bc51d627.png">


#### Testing Instructions



* Visit Tools > Marketing and verify that you don't see the "No Ads" addon upsell. 
* Verify that you see the Premium plan nudge.
* Click on the upsell nudge and verify that it takes you to `/plans` with the "Remove WordPress.com Ads" feature highlighted.
* In non-EN, verify that you don't see the upsell nudge till the translations are completed. 
* Verify mobile view looks fine.
